code/log_helper.py

```diff
@@ -0,0 +1,17 @@
+import os
+import sys
+import logging
+
+def getLogger(name):
```

Comment:
Super nitpicky, but PEP8 python function naming convention would be get_logger().

Reply:
Happy to change this. I went back and forth on it, but thought it might be more user friendly to copy the existing `logging.getLogger()` name.

Reply:
Ah, good point. I like that reasoning. Let's leave it.